### ISSUE TITLE
[BUG] CRC 2.22.1  Install just runs forever

### ISSUE THREAD

**Issue by garyd2** [crc-org/crc#3745]
### General information

  * OS: Linux / macOS / Windows = MacOS
  * Hypervisor: KVM / Hyper-V / hyperkit / vfkit
  * Did you run `crc setup` before starting it (Yes/No)?  Yes
  * Running CRC on: Laptop / Baremetal-Server / VM   Laptop (not an M1 or M2)

### Steps to reproduce
![Screenshot 2023-06-29 at 15 46 44](https://github.com/crc-org/crc/assets/114586171/5245cc96-5581-44b4-85bf-880f754a78a4)

  1.  Double click the setup
  2. Add the secret from openshift local page
  3. Click next
  4. Install sits like this for hours and hours

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
Can you please try to run `crc setup` from a terminal?

One of the reasons this might 'hang' is that the download phase is slow or impeded.

**Comment by garyd2** [comment]
I can do that yes, is there any plans to add some logging or feedback to the end user of what its doing and failing at, from an end user perspective you just assume its `working away` Thanks

**Comment by gbraad** [comment]
The tray (which serves the onboarding part of the current installer) is to be deprecated. It has not seen any improvments for the last half year. It was supposed to have more details, as can be seen here: https://github.com/crc-org/tray-electron/issues/114

We instead now offer the installation via Podman Desktop with our own extension.

  * https://podman-desktop.io
  * https://github.com/crc-org/crc-extension

Note: the extension is featured, so you do not have to go through the manual installation steps. We are in the process to make the CRC/OpenShift local installer more tailored towards this goal, but this might take some time. It already does not offer the tray as the default installation option.

**Comment by alanfx** [comment]
I have tried running `crc setup` in a terminal, and this is the output:

```
➜  trustification git:(main) ✗ crc setup
INFO Using bundle path /Users/afield/.crc/cache/crc_microshift_vfkit_4.13.3_arm64.crcbundle 
INFO Checking if running as non-root              
INFO Checking if crc-admin-helper executable is cached 
INFO Checking if running on a supported CPU architecture 
INFO Checking if crc executable symlink exists    
INFO Creating symlink for crc executable          
INFO Checking minimum RAM requirements            
INFO Checking if running emulated on a M1 CPU     
INFO Checking if vfkit is installed               
INFO Checking if CRC bundle is extracted in '$HOME/.crc' 
INFO Checking if /Users/afield/.crc/cache/crc_microshift_vfkit_4.13.3_arm64.crcbundle exists 
INFO Checking if old launchd config for tray and/or daemon exists 
INFO Checking if crc daemon plist file is present and loaded 
INFO Adding crc daemon plist file and loading it  
Temporary error: daemon is not running yet (x8)
➜  trustification git:(main) ✗ crc setup
INFO Using bundle path /Users/afield/.crc/cache/crc_microshift_vfkit_4.13.3_arm64.crcbundle 
INFO Checking if running as non-root              
INFO Checking if crc-admin-helper executable is cached 
INFO Checking if running on a supported CPU architecture 
INFO Checking if crc executable symlink exists    
INFO Checking minimum RAM requirements            
INFO Checking if running emulated on a M1 CPU     
INFO Checking if vfkit is installed               
INFO Checking if CRC bundle is extracted in '$HOME/.crc' 
INFO Checking if /Users/afield/.crc/cache/crc_microshift_vfkit_4.13.3_arm64.crcbundle exists 
INFO Checking if old launchd config for tray and/or daemon exists 
INFO Checking if crc daemon plist file is present and loaded 
INFO Adding crc daemon plist file and loading it  
Your system is correctly setup for using CRC. Use 'crc start' to start the instance
➜  trustification git:(main) ✗ crc start
INFO Checking if running as non-root              
INFO Checking if crc-admin-helper executable is cached 
INFO Checking if running on a supported CPU architecture 
INFO Checking if crc executable symlink exists    
INFO Checking minimum RAM requirements            
INFO Checking if running emulated on a M1 CPU     
INFO Checking if vfkit is installed               
INFO Checking if old launchd config for tray and/or daemon exists 
INFO Checking if crc daemon plist file is present and loaded 
INFO Loading bundle: crc_microshift_vfkit_4.13.3_arm64... 
CRC requires a pull secret to download content from Red Hat.
You can copy it from the Pull Secret section of https://console.redhat.com/openshift/create/local.
? Please enter the pull secret ********************************************************************************************************
INFO Creating CRC VM for MicroShift 4.13.3...     
INFO Generating new SSH key pair...               
INFO Starting CRC VM for microshift 4.13.3...     
INFO CRC instance is running with IP 127.0.0.1    
INFO CRC VM is running                            
INFO Updating authorized keys...                  
INFO Configuring shared directories               
INFO Check internal and public DNS query...       
INFO Check DNS query from host...                 
INFO Starting Microshift service... [takes around 1min] 
```

I left this running in a terminal overnight, and no progress is made after the last line is displayed. Here is the version info:

```
➜  trustification git:(main) ✗ crc version
CRC version: 2.23.0+ddcfe8
OpenShift version: 4.13.3
Podman version: 4.4.4
```

**Comment by praveenkumar** [comment]
@alanfx Did you check `crc status` on a different terminal or it is hanged because of hibernate? Did you try to rerun the process and it again stuck around same step?

**Comment by gbraad** [comment]
@alanfx can you follow-up? What does `crc status` report for you?

The reported issue was for the UI, but this is a download phase.
From the CLI I can see that was resolved and you were now waiting for the start to finish.

(From the UI, we only start from the tray, so the reported issue is not the same as the follow up message) => If the issue persists, please file a new issue

I suggest closing this.
